http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala

```diff
@@ -148,7 +149,7 @@ class ConfigReaderSpec extends AnyFlatSpec with Matchers {
         ),
         CromwellRunnerAppConfig(
           ChartName("terra-helm/cromwell-runner-app"),
-          ChartVersion("0.164.0"),
+          ChartVersion("0.167.0"),
```

Comment:
wondering if this is also a necessary part of the PR

Reply:
Yep!  It goes with the other change - this test fails without it.  If the latest CRA chart gets merged in before this PR does, then I won't need it anymore, but until then, necessary.